README.md

```diff
@@ -66,6 +66,8 @@ In order to start up oneview-redfish-toolkit service, there is some mandatory co
 
   * **xml_prettify**: whether XML objects on answers are indented or not
 
+  * **host**: the IP address where redfish service will listen to
```

Comment:
explain that 0.0.0.0 means listen too all ip address but it dont work on windows

Reply:
Done!